errors_test.go

```diff
@@ -146,6 +146,25 @@ func TestHiddenWithCause(t *testing.T) {
 	// doesn't panic. If we get to this point without panicking, we're happy.
 }
 
+func TestFill(t *testing.T) {
+	e := New("something happened").(*baseError)
+	e2 := New("uh oh: %v", e).(*wrappingError)
```

Comment:
Might be nice to include a standard Go error in the chain to make sure that keeps working.

Reply:
That's smart.  Just added that.